contracts/identitytreestore/IdentityTreeStore.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: GPL-3.0
-pragma solidity 0.8.16;
+pragma solidity 0.8.20;
```

Comment:
can we update pragma for already deploy contract? will it have influence on upgrade? (for each contract)

Reply:
It shouldn't. The version is minor upgrade so no breaking changes and different behaviour in logic expected. Most important, storage layout will not be changed for sure.